src/box/lua/config/instance_config.lua

```diff
@@ -1118,6 +1118,60 @@ return schema.new('instance_config', schema.record({
             end
         end,
     }),
+    security = schema.record({
+        auth_type = schema.enum({
+            'chap-sha1',
+            'pap-sha256',
+        }, {
+            box_cfg = 'auth_type',
+            default = 'chap-sha1',
+        }),
+        auth_delay = schema.scalar({
+            type = 'number',
+            default = 0,
+            box_cfg = 'auth_delay',
+        }),
```

Comment:
This option is supported only on Tarantool Enterprise Edition. We should reflect this fact somehow. Please, check the `enterprise_edition()` fucnction, which should handle it appropriately.

The same for other options of the section.

(I guess it should resolve the test failes in CI.)